### PR TITLE
Add PouchDB nuxt module

### PR DESCRIPTION
**Pull request by buhrmi**
Been working on a pouchdb integration for vue. Extracted it into a nuxt module. planning to keep working on it and improve documentation and configuration.

### CODE REVIEW

File: modules/pouchdb/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "homepage": "https://github.com/nuxt/modules/modules/pouchdb",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "private": false,
+  "author": "Stefan Buhrmester <buhrmi@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "pouchdb": "*",
```
Comment (pi0) [inline modules/pouchdb/package.json]: Would you please prefer caret version dependencies (`^x.y.x`)? `*` can break dependent packages on pouchdb's major bumps.

File: modules/pouchdb/plugin.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import VuePouch from 'vue-pouch'
+import PouchDB from 'pouchdb'
+import Find from 'pouchdb-find'
+import LiveFind from 'pouchdb-live-find'
```
Comment (pi0) [inline modules/pouchdb/plugin.js]: Isn't necessary documenting of `find` and `live-find` existence with the module?

**Comment by pi0** [comment]
Awesome job @buhrmi so far +1. Are you interested in further works on this module? Seems some missing features:
- Ability to define remote replicate DB
- Enable/Disable debug mode
- Providing more options to Constructor

File: modules/pouchdb/plugin.js (hunk 2 above)
Comment (buhrmi) [inline modules/pouchdb/plugin.js]: thanks for the quick review. done.

**Comment by buhrmi** [comment]
Yeah I'm going to work on those features and support for SSR

**Comment by buhrmi** [comment]
will keep this module in it's own repo

**Pull request closed without merging**